src/nft/hooks/useLazyEffect.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useRef } from 'react'
+
+export function useLazyEffect(fn: () => void, inputs: unknown[]) {
```

Comment:
Document what this function does.
This can probably be moved to src/hooks, as it is a general utility.